src/components/RichSelect/index.tsx

```diff
@@ -380,14 +421,14 @@ const ValueContainer = ({
   </components.ValueContainer>
 )
 
-ValueContainer.defaultProps = {
-  children: null,
-  hasValue: false,
-  isDisabled: false,
-  isMulti: false,
-  noTopLabel: false,
-  selectProps: {},
-}
+// ValueContainer.defaultProps = {
+//   children: null,
+//   hasValue: false,
+//   isDisabled: false,
+//   isMulti: false,
+//   noTopLabel: false,
+//   selectProps: {},
+// }
```

Comment:
You can set default values to `ValueContainer` function declaration and remove `defaultProps` here